.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,40 @@
-## About
+## Pull Request Type
+<!-- Check the appropriate option -->
 
-This pull request is about (X), which does (Y).
+- [ ] Feature addition
+- [ ] Bugfix
+- [ ] Documentation update
+- [ ] Code refactor
+- [ ] Tests improvement
+- [ ] CI/CD pipeline enhancement
+- [ ] Other: [Replace with a description]
 
-## Checklist
+## Description
+<!-- Provide a clear and concise description of the purpose of this PR and why it should be merged -->
+<!-- If your code adds or changes features, a usage example would be helpful -->
+
+
+## Changes
+<!-- List the changes you have made in a bullet-point format -->
 
-- [ ] The ``pre-commit`` code linter has been run over all edited files to ensure the code is linted.
-- [ ] I've ensured the change(s) work on `3.8.6` and higher.
-- [ ] I have added the `versionadded`, `versionchanged` and `deprecated` to any new or changed user-facing function I committed.
-<!-- If you are unsure what the next version is, feel free to ask in #unstable at https://discord.gg/interactions -->
 
-### Pull-Request specification
+## Related Issues
+<!-- If this PR is related to any open issues, please mention them using "#ISSUENUMBER" -->
 
-I've made this pull request: (check all that apply)
-  - [ ] For the documentation
-  - [ ] To add a new feature
-  - [ ] As a general enhancement
-  - [ ] As a refactor of the library/the library's code
-  - [ ] To fix an existing bug
-  - [ ] To resolve #ISSUENUMBER
 
+## Test Scenarios
+<!-- Provide clear instructions on how to test your changes, where applicable - if no tests are required, explain why -->
 
-This is:
-  - [ ] A breaking change
 
-<!--- Expand this when more comes up--->
+## Python Compatibility
+<!-- Testing 3.11 is not strictly required, but it would be nice if you could confirm that it works. -->
+- [ ] I've ensured my code works on Python `3.10.x`
+- [ ] I've ensured my code works on Python `3.11.x`
+
+
+## Checklist
+<!-- If you have not completed all of the following, there is a good chance your PR will not be merged. -->
+- [ ] I've run the `pre-commit` code linter over all edited files
```

Comment:
Do we still want the pre-commit line? Since we have the precommit bot that modifies the code automatically...

Reply:
its good practice to run pre-commit locally, and not rely on the automated CI (note pre **commit** )